src/core/blockchain/celo/config.ts

```diff
@@ -151,6 +151,9 @@ export const config: IBlockchainConfig = {
     defaultOrder: 1,
     amountToKeepInAccount: {
         [AccountType.DEFAULT]: new BigNumber(0)
+    },
+    amountToKeepForTheFees: {
```

Comment:
why are we adding this ? `amountToKeepInAccount` is exactly for fees :)

Reply:
This task: https://app.zenhub.com/workspaces/moonlet-wallet-5d93018da1bdb00001e54a59/issues/moonlet/wallet-app/1206
 says to check for 1 zil. The amountToKeepInAccount is 100 zil.

Reply:
I should probably rename it